### ISSUE TITLE
there is no search "parameters" contents on the swagger part in fhir-swagger-generator-4.10.0-cli.jar

### ISSUE THREAD

**Issue by cpeter0702** [LinuxForHealth/FHIR#3821]
**Describe the bug**
I can generate Patient-swagger.json and see the search "parameters" contents in the paths./Patient.get.parameters through com.ibm.fhir.swagger.generator.FHIRSwaggerGenerator in fhir-swagger-generator-4.9.2-cli.jar
However, I cannot see any contents in the fhir-swagger-generator-4.10.2-cli.jar

**Environment**
Which version of IBM FHIR Server?
4.10.2

**To Reproduce**
1. Go to [https://github.com/LinuxForHealth/FHIR/releases](https://github.com/LinuxForHealth/FHIR/releases)
2. Download [fhir-swagger-generator-4.10.2-cli.jar](https://github.com/LinuxForHealth/FHIR/releases/download/4.10.2/fhir-swagger-generator-4.10.2-cli.jar)
3. Use command line to execute java -cp fhir-swagger-generator-4.10.2-cli.jar com.ibm.fhir.swagger.generator.FHIRSwaggerGenerator
4. Go into the folder swagger that just generated and open an example like Patient-swagger.json
5. You cannot see any contents in the paths.'/Patient'.get.parameters

**Expected behavior**
I expect it to have the same contents as version 4.9.2.

"parameters": [
          {
            "name": "given",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A portion of the given name of the patient\r\n* [Practitioner](practitioner.html): A portion of the given name\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "identifier",
            "description": "A patient identifier",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "birthdate",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): The patient's date of birth\r\n* [Person](person.html): The person's date of birth\r\n* [RelatedPerson](relatedperson.html): The Related Person's date of birth\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "address",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A server defined search that may match any of the string fields in the Address, including line, city, district, state, country, postalCode, and/or text\r\n* [Person](person.html): A server defined search that may match any of the string fields in the Address, including line, city, district, state, country, postalCode, and/or text\r\n* [Practitioner](practitioner.html): A server defined search that may match any of the string fields in the Address, including line, city, district, state, country, postalCode, and/or text\r\n* [RelatedPerson](relatedperson.html): A server defined search that may match any of the string fields in the Address, including line, city, district, state, country, postalCode, and/or text\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "deceased",
            "description": "This patient has been marked as deceased, or as a death date entered",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "address-state",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A state specified in an address\r\n* [Person](person.html): A state specified in an address\r\n* [Practitioner](practitioner.html): A state specified in an address\r\n* [RelatedPerson](relatedperson.html): A state specified in an address\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "gender",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): Gender of the patient\r\n* [Person](person.html): The gender of the person\r\n* [Practitioner](practitioner.html): Gender of the practitioner\r\n* [RelatedPerson](relatedperson.html): Gender of the related person\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "general-practitioner",
            "description": "Patient's nominated general practitioner, not the organization that manages the record",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "link",
            "description": "All patients linked to the given patient",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "active",
            "description": "Whether the patient record is active",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "language",
            "description": "Language code (irrespective of use value)",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "address-postalcode",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A postalCode specified in an address\r\n* [Person](person.html): A postal code specified in an address\r\n* [Practitioner](practitioner.html): A postalCode specified in an address\r\n* [RelatedPerson](relatedperson.html): A postal code specified in an address\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "address-country",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A country specified in an address\r\n* [Person](person.html): A country specified in an address\r\n* [Practitioner](practitioner.html): A country specified in an address\r\n* [RelatedPerson](relatedperson.html): A country specified in an address\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "death-date",
            "description": "The date of death has been provided and satisfies this search value",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "phonetic",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A portion of either family or given name using some kind of phonetic matching algorithm\r\n* [Person](person.html): A portion of name using some kind of phonetic matching algorithm\r\n* [Practitioner](practitioner.html): A portion of either family or given name using some kind of phonetic matching algorithm\r\n* [RelatedPerson](relatedperson.html): A portion of name using some kind of phonetic matching algorithm\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "phone",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A value in a phone contact\r\n* [Person](person.html): A value in a phone contact\r\n* [Practitioner](practitioner.html): A value in a phone contact\r\n* [PractitionerRole](practitionerrole.html): A value in a phone contact\r\n* [RelatedPerson](relatedperson.html): A value in a phone contact\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "organization",
            "description": "The organization that is the custodian of the patient record",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "name",
            "description": "A server defined search that may match any of the string fields in the HumanName, including family, give, prefix, suffix, suffix, and/or text",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "address-use",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A use code specified in an address\r\n* [Person](person.html): A use code specified in an address\r\n* [Practitioner](practitioner.html): A use code specified in an address\r\n* [RelatedPerson](relatedperson.html): A use code specified in an address\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "telecom",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): The value in any kind of telecom details of the patient\r\n* [Person](person.html): The value in any kind of contact\r\n* [Practitioner](practitioner.html): The value in any kind of contact\r\n* [PractitionerRole](practitionerrole.html): The value in any kind of contact\r\n* [RelatedPerson](relatedperson.html): The value in any kind of contact\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "address-city",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A city specified in an address\r\n* [Person](person.html): A city specified in an address\r\n* [Practitioner](practitioner.html): A city specified in an address\r\n* [RelatedPerson](relatedperson.html): A city specified in an address\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "family",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A portion of the family name of the patient\r\n* [Practitioner](practitioner.html): A portion of the family name\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "name": "email",
            "description": "Multiple Resources: \r\n\r\n* [Patient](patient.html): A value in an email contact\r\n* [Person](person.html): A value in an email contact\r\n* [Practitioner](practitioner.html): A value in an email contact\r\n* [PractitionerRole](practitionerrole.html): A value in an email contact\r\n* [RelatedPerson](relatedperson.html): A value in an email contact\r\n",
            "in": "query",
            "type": "string",
            "required": false
          },
          {
            "$ref": "#/parameters/_profileParam"
          },
          {
            "$ref": "#/parameters/_lastUpdatedParam"
          },
          {
            "$ref": "#/parameters/_tagParam"
          },
          {
            "$ref": "#/parameters/_securityParam"
          },
          {
            "$ref": "#/parameters/_sourceParam"
          },
          {
            "$ref": "#/parameters/_idParam"
          }
        ],

**Additional context**

- You can download the file fhir-swagger-generator-4.9.2-cli.jar [here](https://github.com/LinuxForHealth/FHIR/releases/download/4.9.2/fhir-swagger-generator-4.9.2-cli.jar)
- You can download the file fhir-swagger-generator-4.10.2-cli.jar [here](https://github.com/LinuxForHealth/FHIR/releases/download/4.10.2/fhir-swagger-generator-4.10.2-cli.jar)
- thank you very much if you can help me to resolve this issue

**Comment by lmsurpre** [comment]
I confirmed that this is an issue in our latest release (4.11.1) as well as in main.
I think its a side-effect from when we changed the way we load the Search Parameters...we look them up from our internal registry now but the registry project was never added as a dependency here.
It works as expected when I run from eclipse (since the registry is on my classpath) but in the all-in-one executable cli jar it does not (because the registry--and thus the SearchParameter definitions--are not there).

I think the fix is pretty simple...we just need to add the `fhir-registry` component to the classpath.
And in `main`, we need to add `fhir-core-r4b` as well.
